docs/components/messageListener.md

```diff
@@ -0,0 +1,168 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# `gamepad.messageListener`
+
+This component listens to the messages and signals sent by the [`inputMapper`](inputMapper.md) component in the content
+scripts and then calls the action invokers according to the received message.
+
+## Using this grade
+
+The component can be used by creating its instance or by making a custom component using the grade.
+
+``` javascript
+// Either create an instance of the messageListener.
+var messageListenerInstanceOne = gamepad.messageListener();
+
+// Otherwise create a custom component using the messageListener grade.
+fluid.defaults("my.messageListener.grade", {
+    gradeNames: ["gamepad.messageListener", "my.CustomGrade"]
+});
+var messageListenerInstanceTwo = my.messageListener.grade();
+```
+
+## Component Options
+
+This component does not provide any configuration options.
+
+## Non-navigation Invokers
+
+### `{messageListener}.addListener()`
+
+- Returns: Nothing.
+
+Listens to the messages received from the [`inputMapper`](inputMapper.md) component in the content scripts and triggers
+the [actionExecutor](#messagelisteneractionexecutoractiondata) invoker as a callback function.
+
+### `{messageListener}.actionExecutor(actionData)`
+
+- `actionData {Object}` The message object received from the [`inputMapper`](inputMapper.md) component in the content
+  scripts.
+- Returns: Nothing.
+
+Identifies and triggers the **navigation-producing invokers** according to the `actionData`. The navigation-producing
+invoker is triggered only if the component provides it. Each of these invokers accepts the following arguments:
+
+- `tabId`: ID of the currently active tab.
+- `invert`: Whether direction of the given action should be in the opposite order (for thumbsticks).
+- `active`: Whether a new tab or window should be focused when opened.
+- `homepageURL`: URL that a new tab or window should load when opened.
+- `left`: Position of the current browser window from the screen's left edge (in pixels).
+
+The invokers use only those arguments required to perform their specific action.
+
+## Navigation Invokers
+
+### `{messageListener}.openNewTab(active, homepageURL)`
+
+- `active {Boolean}` Whether a new browser tab should be active when opened.
```

Comment:
You might want to use a word other than "active" to describe this.  I assume you mean "focused"?  If so, changing the description is enough.